src/include/rados/librados.hpp

```diff
@@ -757,6 +757,21 @@ namespace librados
     int writesame(const std::string& oid, bufferlist& bl,
 		  size_t write_len, uint64_t off);
     int read(const std::string& oid, bufferlist& bl, size_t len, uint64_t off);
+    int repair_read(const std::string& oid, bufferlist& bl, size_t len, uint64_t off, int flags, int32_t osdid, uint32_t epoch);
```

Comment:
not really, the result will be stored to the operation's own rval. so it's still good to use it?